tests/dashboard/guest-management.test.js

```diff
@@ -3,6 +3,7 @@ const common = require('../common'),
 
 describe('Guest Management', () => {
   before(() => common.login(browser, 'admin', 'admin', 'local'));
+  after(() => common.logout());
   it('should navigate to Guest Management', () =>
     common.clickSidebarTab(browser, 'Guest Management'));
 });
```

Comment:
just the setup for testing guest pools will be more complicated than any of my current tests